app.js

```diff
@@ -61,14 +64,13 @@ function checkedTodo(i) {
     finalTodoUpdate(i);
   }, 500);
 }
-function DeleteTodo(item) {
-  let text ="Do you want to delete this item.";
-  if(confirm(text)==true)
-  {
-  let todo = document.getElementById(`item-${item}`);
-  todo.remove();
+function deleteTodo(item) {
+  let text = "Do you want to delete this item.";
```

Comment:
@RiyaBasak26 please explain this codeblock

Reply:
it basically conforming whether we really want delete the item or we have press the delete button by mistake